src/components/AddPoll/addPoll.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
```

Comment:
I still do not know Why I am seeing AddPoll related stuff in Login Branch @PuniaMohit